bundle/bundle.go

```diff
@@ -48,6 +48,9 @@ type Bundle struct {
 	// Exclusively use this field for filesystem operations.
 	SyncRoot vfs.Path
 
+	// Path to root of git worktree
```

Comment:
Can you add a line here explaining what the root of a git worktree is? Or redirect to the docs? Nice to call that out since It's not the same as the root of a git repository.